functions/Remove-DbaDbUser.ps1

```diff
@@ -0,0 +1,230 @@
+                                Write-Message -Level Warning -Message "User owns objects in the database and will not be removed."
+                                foreach ($ownedUrn in $ownedUrns) {
+                                    $obj = $server.GetSmoObject($ownedUrn)
+                                    Write-Message -Level Warning -Message "User $user owns $($obj.GetType().Name) $obj"
+                                }
+                                return  # exit function
+                            }
+                        }
+
+                        # Change the owner of any schema not the same name as the user
+                        if ($schema.Name -NE $user.Name) {
+                            # Check for owned objects early so we can exit before any changes are made
+                            $ownedUrns = $schema.EnumOwnedObjects()
+                            if (($ownedUrns -And $Force) -Or (-Not $ownedUrns)) {
+                                $alterSchemas += $schema
+                            } else {
+                                Write-Message -Level Warning -Message "User $user owns the Schema $schema, which owns $($ownedUrns.Count) Object(s).  If you want to change the schemas' owner to [dbo] and drop the user anyway, use -Force parameter.  User $user will not be removed."
+                                return  #exit the function
+                            }
+                        }
+                    }
+
+                    # Perform the changes with the owner alters first so function can exit w/o a drop if -Force wasn't used
+                    foreach ($schema in $alterSchemas) {
+                        Write-Message -Level Verbose -Message "Owner of Schema $schema will be changed to [dbo]."
+                        if ($PSCmdlet.ShouldProcess($server, "Change the owner of Schema $schema to [dbo].")) {
+                            $schema.Owner = "dbo"
+                            $schema.Alter()
+                        }
+                    }
+
+                    foreach ($schema in $dropSchemas) {
+                        if ($PSCmdlet.ShouldProcess($server, "Drop Schema $schema from Database $db.")) {
+                            $schema.Drop()
+                        }
+                    }
+                }
+
+                # Drop the User
+                if ($PSCmdlet.ShouldProcess($server, "Drop User $user from Database $db.")) {
+                    try {
+                        $user.Drop()
+                        [pscustomobject]@{
+                            SqlInstance = $server.name
+                            Database    = $db.name
+                            User        = $user
+                            Status      = "Dropped"
+                        }
+                    } catch {
+                        Write-Message -Level Verbose -Message "Could not drop $user from Database $db on target $server"
+                        [pscustomobject]@{
```

Comment:
Need to adjust the `SQLInstance` value and then also add the Computer and Instance name:

```powershell
ComputerName             = $server.NetName
InstanceName             = $server.ServiceName
SqlInstance              = $server.DomainInstanceName
```